addons/common/functions/fnc_setBuildingMarker.sqf

```diff
@@ -0,0 +1,64 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Addes or removes a rectangle marker on object bounding box, to make placed building look like terrain building. Called by CBA serverEvent.
+ *
+ * Arguments:
+ * 0: Object <OBJECT>
+ * 1: Set Marker <BOOLEAN>
+ *
+ * Return Value:
+ * Marker name <STRING>
+ *
+ * Example:
+ * [_object, true] call zen_common_fnc_setbuildingMarker
+ *
+ * Public: No
+ */
+
+params ["_object", "_setMarker"];
+
+if !(_object isEqualType objNull) exitWith {""};
```

Comment:
Add type checking to params`.

Reply:
https://github.com/zen-mod/ZEN/pull/704/commits/df7388c87d42a6f409e1256b36478aa186d5e9f1#diff-904878a4c3022a6347e5b85fe824f3641b2911718a3985159b1d6bdf0b3ad5e0R19